### PR TITLE
New firewall rules to allow access  to Nomis

### PR DESCRIPTION
**Pull request by SteveLinden**
## A reference to the issue / Description of it

See #5713 

## How does this PR fix the problem?

Adds the firewalls for when the ports are opened

## How has this been tested?

Not yet testable

## Checklist (check `x` in `[ ]` of list items)

- [ x] I have performed a self-review of my own code
- [ x] All checks have passed

### CODE REVIEW

**Review by dms1981 — CHANGES_REQUESTED** [review]
Just one more comment from me

File: terraform/environments/core-network-services/cidr-ranges.tf [hunk 1]
```diff
@@ -44,6 +44,8 @@ locals {
     nomisapi-t3-root-vnet          = "10.47.0.0/26"
     nomisapi-preprod-root-vnet     = "10.47.0.64/26"
     nomisapi-prod-root-vnet        = "10.47.0.128/26"
+    moj-smtp-relay1              = "10.180.104.100"
```
Comment (dms1981) [inline terraform/environments/core-network-services/cidr-ranges.tf:47]: Missing the `/32` at the end here

File: terraform/environments/core-network-services/cidr-ranges.tf [hunk 2]
```diff
@@ -44,6 +44,8 @@ locals {
...
+    moj-smtp-relay2              = "10.180.105.100"
```
Comment (dms1981) [inline terraform/environments/core-network-services/cidr-ranges.tf:48]: Also missing the `/32` at the end here.

Comment (SteveLinden) [reply terraform/environments/core-network-services/cidr-ranges.tf]: Amended and repushed

File: terraform/environments/core-network-services/cidr-ranges.tf (hunk 1 above)
Comment (SteveLinden) [reply terraform/environments/core-network-services/cidr-ranges.tf]: Amended and repushed

**Review by dms1981 — APPROVED** [review]

**Pull request merged**